pytensor/tensor/blas.py

```diff
@@ -772,27 +772,15 @@ def build_gemm_call(self):
         else:
             setup_z_Nz_Sz = self.setup_z_Nz_Sz
 
-        return "".join(
-            (
-                self.declare_NS,
-                self.check_xyz_rank2,
-                setup_z_Nz_Sz,
-                self.check_xyz_double_or_float,
-                self.check_ab_double_or_float,
-                self.broadcast_xy,
-                self.check_dims,
-                self.check_strides,
-                self.encode_strides_in_unit,
-                self.compute_strides,
-                self.begin_switch_typenum,
-                self.case_float,
-                self.case_float_ab_constants,
-                self.case_float_gemm,
-                self.case_double,
-                self.case_double_ab_constants,
-                self.case_double_gemm,
-                self.end_switch_typenum,
-            )
+        return (
+            f"{self.declare_NS}{self.check_xyz_rank2}{setup_z_Nz_Sz}"
+            f"{self.check_xyz_double_or_float}{self.check_ab_double_or_float}"
+            f"{self.broadcast_xy}{self.check_dims}{self.check_strides}"
+            f"{self.encode_strides_in_unit}{self.compute_strides}"
+            f"{self.begin_switch_typenum}{self.case_float}"
+            f"{self.case_float_ab_constants}{self.case_float_gemm}"
+            f"{self.case_double}{self.case_double_ab_constants}"
+            f"{self.case_double_gemm}{self.end_switch_typenum}"
```

Comment:
The previous version was much easier to read

Reply:
agreed

Reply:
agreed xD